recipes/mlst-cge/python_path.patch

```diff
@@ -0,0 +1,17 @@
+--- a/mlst.py
++++ b/mlst.py
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env python3
++#!/opt/anaconda1anaconda2anaconda3/bin/python
```

Comment:
I think we might be able to remove this one.

Reply:
The environment variable is OK. Its more so changing the shebang line. Here ist should stay as `/usr/bin/env python3`  instead of `/opt/anaconda1anaconda2anaconda3/bin/python` 

In most cases users won't have python in `/opt/anaconda1anaconda2anaconda3/bin`